internal/webhooks/cluster.go

```diff
@@ -268,6 +268,21 @@ func (webhook *Cluster) validateTopology(ctx context.Context, oldCluster, newClu
 				),
 			)
 		}
+		// A +2 minor version upgrade is not allowed.
+		ceilVersion := semver.Version{
+			Major: oldVersion.Major,
+			Minor: oldVersion.Minor + 2,
+			Patch: 0,
+		}
+		if inVersion.GTE(ceilVersion) {
+			allErrs = append(
+				allErrs,
+				field.Forbidden(
+					field.NewPath("spec", "topology", "version"),
+					fmt.Sprintf("cannot update Kubernetes version from %s to %s", oldVersion, inVersion),
```

Comment:
Should we make this message consistent with l.267?
e.g. would be `version cannot be increased from %q to %q`